### ISSUE TITLE
Provide a better explanation for end users of what the maintenance intention badge means

### ISSUE THREAD

**Issue by jhinch** [rust-lang/crates.io#2134]
Currently, if you are browsing crates.io, some crates display a maintenance intention. For example if you look at https://crates.io/crates/lazy_static you'll see a badge 'maintenance: passively-maintained'. When you hover over it, the alt text simple says 'Maintenance intention for this crate' and you cannot click through to any page to see what it means. Most other badges you can click through to something, such as a CI build, to get more information on what the badge means.

What made this worse for me is that I am running Privacy Badger, a popular addon which blocks trackers, protecting users privacy. It turns out that img.shields.io which hosts the image badge uses tracking cookies and as a result, the badge was blocked from loading and the alt text was the only thing which I would see. It took me some time to figure out that this was suppose to be a badge _and_ what the value of the badge actually meant. I managed to find the definitions in an RFC and the Cargo reference documentation

https://github.com/rust-lang/rfcs/blob/master/text/1824-crates.io-default-ranking.md#maintenance
https://doc.rust-lang.org/cargo/reference/manifest.html

I'm proposing that the experience here could be improved. Specifically I think that the alt text could be modified to reference what the maintenance intention actually is (e.g. 'Maintenance intention: passive maintenance') which has the added benefit of assisting people using screen readers. Secondly, I would suggest that the badge should link to the definitions for a more thorough explanation of what the maintenance intention means. This could be in the form of a page similar to https://crates.io/category_slugs where the maintenance intentions are lists and described, or to the Cargo documentation (linking to the RFC probably doesn't make sense as the definitions are only a subsection of a greater RFC).

I would be happy to explore implementing this if the crates.io team agree it is a good idea.

**Comment by carols10cents** [comment]
I think your idea sounds great! Please let us know if you have any questions while implementing!!

**Comment by jhinch** [comment]
I believe this ticket has been addressed now as part of the PR which was merged

**Comment by Turbo87** [comment]
yep, thank you! :)
